test/loadTest/updates_time_measure.sh.sh

```diff
@@ -0,0 +1,92 @@
+﻿#!/bin/bash
+# -*- coding: latin-1 -*-
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# Author: Ivan Arias Leon
+
+# variables
+HOST="localhost"
+PORT="1026"
+NGSI_VERSION="v1"
+UPDATE_TOTAL=100
+SUBSC=false
+SUBSC_REFERENCE="http://localhost:1028/notify"
+
+function random_string(){
+    # a random string is generated with a given size
+    cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w ${1:-32} | head -n 1
```

Comment:
A bit cryptic use of $1

I usually receive the parameters and give them a name before I continue the function.
E.g.:

    # ========================================================
    # 
    # random_string - generate a random string with a given size
    # 
    function random_string()
    {
      size=$1
      cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w ${size:-32} | head -n 1
    }

Something like that. The function gets easier to understand.
Also, the function header makes it easier to browse the code.
You clearly see where a function starts and ends.

NTC